README.md

```diff
@@ -1292,6 +1295,18 @@ app.UseSwaggerUI(c =>
 });
 ```
 
+### Inject Custom JavaScript ###
+
+To tweak the behavior, you can inject additional JavaScript files by adding them to your `wwwroot` folder and specifying the relative paths in the middleware options:
+
+```csharp
+app.UseSwaggerUI(c =>
+{
+    ...
```

Comment:
As otherwise it breaks the C# syntax highlighting

```suggestion
```

Reply:
For me it looks the same as in the existing "Inject Custom CSS" section. But if you want I can remove it. The question is then, should it also be removed in the existing section(s)?

Reply:
Well I only see what's in the diff 😄

I would remove it from all so all the snippets are valid.

Reply:
Ok, done. At least for the Swagger UI code snippets, because I assumed that's what you meant. ("All" would mean over 40 occurences of "...".)

You can prevew it here: https://github.com/cremor/Swashbuckle.AspNetCore/tree/patch-1?tab=readme-ov-file#inject-custom-javascript